src/chatdbg/chatdbg_lldb.py

```diff
@@ -12,6 +12,7 @@
 
 from assistant.lite_assistant import LiteAssistant
 import chatdbg_utils
+import clangd_lsp_integration
```

Comment:
Add to `pyproject.toml`; probably need to add error handling / messages if clangd is not installed

Reply:
This is our own file (`clangd_lsp_integration.py`), currently if clangd is not present it'll be some subprocess open error. I can wrap that and output a better message.

A bigger issue can possibly be if clangd crashes during execution, then I'm not sure what happens. Not sure how common that is though.